backend/routes/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const path = require('path');
+const router = express.Router();
+
+router.post('/api/postRefer', (req, res, next) => {
+  const post = req.body;
+  console.log(post);
+  res.status(201).json({
+    message: 'Post added successfully'
+  });
+});
+
+router.get('/api/getRefer', (req, res, next) => {
+  const refer = [
+    {
+      id: 'fd345',
+      title: 'Some title',
+      content: 'some content'
+    },
+    {
+      id: 'ed345',
+      title: 'Some title2',
+      content: 'new content'
+    }
+  ];
+  res.status(200).json({
+    message: 'get successfully',
+    posts: refer
+  });
+});
+
+router.use('/sandbox', require('./sandbox'));
+//experimenting
```

Comment:
can remove the comment experimenting...

Reply:
done in https://github.com/armory-uci/vital/pull/3/commits/69f3fa19f309c5fcd18dfc9e48e80b5451bfebbf